test/kubernetes/integration_test.go

```diff
@@ -246,6 +246,138 @@ func TestDeployment_Single_Volume(t *testing.T) {
+					VolumeMounts: []v1.VolumeMount{
+						{
+							MountPath: "/data",
+							Name:      volumeName,
+						},
+					},
+					Command: []string{
+						"sleep",
+						"1000000",
+					},
+				},
+			},
+			Volumes: []v1.Volume{
+				{
+					Name: volumeName,
+					VolumeSource: v1.VolumeSource{
+						PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
+							ClaimName: claimName,
+						},
+					},
+				},
+			},
+		},
+	}
+
+	t.Log("Creating pod")
+	_, err := client.CoreV1().Pods(namespace).Create(pod)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	pvc := &v1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: claimName,
+		},
+		Spec: v1.PersistentVolumeClaimSpec{
+			AccessModes: []v1.PersistentVolumeAccessMode{
+				v1.ReadWriteOnce,
+			},
+			Resources: v1.ResourceRequirements{
+				Requests: v1.ResourceList{
+					v1.ResourceStorage: resource.MustParse("5Gi"),
+				},
+			},
+			StorageClassName: strPtr(testStorageClass),
+		},
+	}
+
+	t.Log("Creating pvc")
+	_, err = client.CoreV1().PersistentVolumeClaims(namespace).Create(pvc)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Logf("Waiting pod %q to be running ...\n", pod.Name)
+	if err := waitForPod(client, pod.Name); err != nil {
+		t.Error(err)
+	}
+
+	createdPVC, err := client.CoreV1().PersistentVolumeClaims(namespace).Get(claimName, metav1.GetOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// get pv name
+	pvName := createdPVC.Spec.VolumeName
+	pv, err := client.CoreV1().PersistentVolumes().Get(pvName, metav1.GetOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if pv.Spec.Capacity["storage"] != resource.MustParse("5Gi") {
+		t.Fatal("initial volume size is not equal to requested volume size")
+	}
+
+	t.Log("Updating pvc to request more size")
+	createdPVC.Spec.Resources.Requests = v1.ResourceList{
+		v1.ResourceStorage: resource.MustParse("6Gi"),
+	}
+
+	updatedPVC, err := client.CoreV1().PersistentVolumeClaims(namespace).Update(createdPVC)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Logf("Waiting for volume %q to be resized ...\n", pvName)
+	if err := waitForVolumeCapacityChange(client, pvName, pv.Spec.Capacity); err != nil {
+		t.Error(err)
+	}
+
+	resizedPv, err := client.CoreV1().PersistentVolumes().Get(pvName, metav1.GetOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if resizedPv.Spec.Capacity["storage"] != resource.MustParse("6Gi") {
+		t.Fatal("volume size is not equal to requested volume size")
+	}
+
+	t.Logf("Waiting for volume claim %q to be resized ...\n", claimName)
+	if err := waitForVolumeClaimCapacityChange(client, claimName, updatedPVC.Status.Capacity); err != nil {
+		t.Error(err)
+	}
+
+	resizedPVC, err := client.CoreV1().PersistentVolumeClaims(namespace).Get(claimName, metav1.GetOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if resizedPVC.Status.Capacity["storage"] != resource.MustParse("6Gi") {
+		t.Fatal("file system resize failed: claim status capacity is not equal to requested capacity")
+	}
+
+	t.Log("Finished!")
```

Comment:
`t.Log()` statements only emit messages when a test fails, so we can drop this one.